crates/biome_cli/examples/text_reporter.rs

```diff
@@ -1,26 +1,21 @@
 use biome_cli::{
     DiagnosticsPayload, Execution, Reporter, ReporterVisitor, TraversalSummary, VcsTargeted,
 };
-use biome_service::projects::ProjectKey;
 
 /// This will be the visitor, which where we **write** the data
 struct BufferVisitor(String);
 
 /// This is the reporter, which will be a type that will hold the information needed to the reporter
 struct TextReport {
-    project_key: ProjectKey,
     summary: TraversalSummary,
 }
 
 impl Reporter for TextReport {
     fn write(self, visitor: &mut dyn ReporterVisitor) -> std::io::Result<()> {
-        let execution = Execution::new_format(
-            self.project_key,
-            VcsTargeted {
-                staged: false,
-                changed: false,
-            },
-        );
+        let execution = Execution::new_format(VcsTargeted {
+            staged: false,
+            changed: false,
+        });
```

Comment:
I really like this change, because it felt weird that `project_key` needed to be part of `Execution`.